### PR TITLE
Update classic theme version to 2.1.0

### PR DESCRIPTION
**Pull request by templatin**
------------------------------------------------------------------------------>

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update theme version
| Type?             | new feature
| BC breaks?        | No
| Deprecations?     | No
| Fixed ticket?     | None
| How to test?      | Go to BO > Design > Theme & Logo page and check that the theme version is updated.
| Possible impacts? | None

### CODE REVIEW

**Comment by kpodemski** [comment]
Thanks @templatin 

I think we should target the next version. A person who did the release forgot to update this file and now it's too late. It should be 2.0.8

**Comment by templatin** [comment]
Exactly, I thought the same thing.

**Review by 0x346e3730 — APPROVED** [review]

File: config/theme.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: classic
 display_name: Classic
-version: 2.0.6
+version: 2.0.8
```
Comment (matks) [inline config/theme.yml]:
```suggestion
version: 2.1.0
```

**Review by matks — COMMENTED** [review]
I checked the PR merged in `develop` and some are not bugfixes -> next version must be a minor, not a patch.

 I update your PR and merge it ;)

**Review by matks — APPROVED** [review]

**Pull request merged**